packages/xstate-store/src/atom.ts

```diff
@@ -39,6 +39,31 @@ const [
 
 let activeSub: Subscriber | undefined = undefined;
 
+type AsyncAtomState<Data, Error = unknown> =
+  | { status: 'pending' }
+  | { status: 'done'; data: Data }
+  | { status: 'error'; error: Error };
```

Comment:
This now doesn't match what is documented by the changeset (and by test titles)

Reply:
Fixed the changeset